www/include/eventLogs/export/Formatter.php

```diff
@@ -0,0 +1,401 @@
+    private const ACKNOWLEDGMENT_MESSAGE_TYPE = 'ACK';
+    private const INITIAL_STATE_VALUE = 'INITIAL STATE';
+    private const NOTIFICATION_TYPE_VALUE = 'NOTIF';
+    private array $hosts = [];
+    private array $serviceStatuses = ['0' => 'OK', '1' => 'WARNING', '2' => 'CRITICAL', '3' => 'UNKNOWN'];
+    private array $hostStatuses = ['0' => 'UP', '1' => 'DOWN', '2' => 'UNREACHABLE',];
+    private array $notificationTypes = ['1' => 'HARD', '0' => 'SOFT'];
+    private int $start = 0;
+    private int $end = 0;
+    private string $notification = '';
+    private string $alert = '';
+    private string $error = '';
+    private string $up = '';
+    private string $down = '';
+    private string $unreachable = '';
+    private string $ok = '';
+    private string $warning = '';
+    private string $critical = '';
+    private string $unknown = '';
+
+    /**
+     * @param int $start
+     * @return void
+     */
+    public function setStart(int $start): void
+    {
+        $this->start = $start;
+    }
+
+    /**
+     * @param int $end
+     * @return void
+     */
+    public function setEnd(int $end): void
+    {
+        $this->end = $end;
+    }
+
+    /**
+     * @param string $notification
+     * @return void
+     */
+    public function setNotification(string $notification): void
+    {
+        $this->notification = $notification;
+    }
+
+    /**
+     * @param string $alert
+     * @return void
+     */
+    public function setAlert(string $alert): void
+    {
+        $this->alert = $alert;
+    }
+
+    /**
+     * @param string $error
+     * @return void
+     */
+    public function setError(string $error): void
+    {
+        $this->error = $error;
+    }
+
+    /**
+     * @param string $up
+     * @return void
+     */
+    public function setUp(string $up): void
+    {
+        $this->up = $up;
+    }
+
+    /**
+     * @param string $down
+     * @return void
+     */
+    public function setDown(string $down): void
+    {
+        $this->down = $down;
+    }
+
+    /**
+     * @param string $unreachable
+     * @return void
+     */
+    public function setUnreachable(string $unreachable): void
+    {
+        $this->unreachable = $unreachable;
+    }
+
+    /**
+     * @param string $ok
+     * @return void
+     */
+    public function setOk(string $ok): void
+    {
+        $this->ok = $ok;
+    }
+
+    /**
+     * @param string $warning
+     * @return void
+     */
+    public function setWarning(string $warning): void
+    {
+        $this->warning = $warning;
+    }
+
+    /**
+     * @param string $critical
+     * @return void
+     */
+    public function setCritical(string $critical): void
+    {
+        $this->critical = $critical;
+    }
+
+    /**
+     * @param string $unknown
+     * @return void
+     */
+    public function setUnknown(string $unknown): void
+    {
+        $this->unknown = $unknown;
+    }
+
+    /**
+     * @param array $hosts
```

Comment:
```suggestion
     * @param array $hosts
```
specify the type of data in the array